2023-06-16 Kwindla Hultman Kramer (#218)
rustacean-station/rustacean-station.org: _episodes/interviews/2023-06-16-kwindla-hultman-kramer.md

---
title: "Daily with Kwindla Hultman Kramer"

> date: 2023-06-16T15:06:00Z

Reviewer: ```suggestion
date: 2023-06-16T12:31:00Z
```

file: https://audio.rustacean-station.org/file/rustacean-station/2023-06-16-kwindla-hultman-kramer.mp3
duration: "01:02:53"
length: "60368146"
#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
---

> Allen Wyma talks with [Kwindla Hultman Kramer](https://www.linkedin.com/in/kwkramer), Found and CEO of [Daily](https://www.daily.co/), and [João Neves](https://www.linkedin.com/in/jo%C3%A3o-neves-8a990411/), Staff Engineer at Daily. Daily is a platform with SDKs that help developers build better video experiences built upon the WebRTC standard using Rust.

Reviewer: ```suggestion
Allen Wyma talks with [Kwindla Hultman Kramer](https://www.linkedin.com/in/kwkramer), Founder and CEO of [Daily](https://www.daily.co/), and [João Neves](https://www.linkedin.com/in/jo%C3%A3o-neves-8a990411/), Staff Engineer at Daily. Daily provides SDKs for building video applications on top of the WebRTC standard using Rust.
```
